test/test_ops.py

```diff
@@ -339,11 +350,45 @@ def test_out(self, device, dtype, op):
         sample = samples[0]
         # call it normally to get the expected result
         expected = op(*sample.input, *sample.args, **sample.kwargs)
-        # call it with out=... and check we get the expected result
-        out_kwargs = sample.kwargs.copy()
-        out_kwargs['out'] = out = torch.empty_like(expected)
-        op(*sample.input, *sample.args, **out_kwargs)
-        self.assertEqual(expected, out)
+
+        def _test(tested_op):
+            # call it with out=... and check we get the expected result
+            out_kwargs = sample.kwargs.copy()
+            out_kwargs['out'] = out = torch.empty_like(expected)
+            tested_op(*sample.input, *sample.args, **out_kwargs)
+            self.assertEqual(expected, out)
+
+        _test(op)
+        for a_op in op.aliases:
+            _test(a_op)
+
+    @ops([op for op in op_db if op.aliases])
+    def test_aliases_jit_name_remapping(self, device, dtype, op):
```

Comment:
@mruberry I replaced jit consistency by jit_name_remapping. Please, let me know what do you think. Thanks

Reply:
Sounds great!